### DISCUSSION TITLE
'DBMS_LOGMNR.NEW' must be declared???

### DISCUSSION THREAD

**Discussion by iamjavadev1** [averemee-si/oracdc#46] [Q&A] [answered]
Hi averemee-si, 

            I I have configured it as you suggested. And when I found out that the connector was registered, but I got the following problem. Have I done something wrong? please help me.
------------------------------------------------------------------------------------------------------------
[2023-08-08 11:01:02,396] ERROR [oracdc-test01|task-0] Unable to start logminer archivelog worker thread! (solutions.a2.cdc.oracle.OraCdcLogMinerWorkerThread:286)
[2023-08-08 11:01:02,397] ERROR [oracdc-test01|task-0] ORA-06550: line 5, column 17:
PLS-00201: identifier 'DBMS_LOGMNR.NEW' must be declared
ORA-06550: line 5, column 5:
PL/SQL: Statement ignored
ORA-06550: line 7, column 17:
PLS-00201: identifier 'DBMS_LOGMNR.ADDFILE' must be declared
ORA-06550: line 7, column 5:
PL/SQL: Statement ignored
ORA-06550: line 9, column 3:
PLS-00201: identifier 'DBMS_LOGMNR.ADD_LOGFILE' must be declared
ORA-06550: line 9, column 3:
PL/SQL: Statement ignored

java.sql.BatchUpdateException: ORA-06550: line 5, column 17:
PLS-00201: identifier 'DBMS_LOGMNR.NEW' must be declared
ORA-06550: line 5, column 5:
PL/SQL: Statement ignored
ORA-06550: line 7, column 17:
PLS-00201: identifier 'DBMS_LOGMNR.ADDFILE' must be declared
ORA-06550: line 7, column 5:
PL/SQL: Statement ignored
ORA-06550: line 9, column 3:
PLS-00201: identifier 'DBMS_LOGMNR.ADD_LOGFILE' must be declared
ORA-06550: line 9, column 3:
PL/SQL: Statement ignored

Thanks in advanced.
Iamjavadev1

**Comment by averemee-si** [comment]
Hi Iamjavadev1,

Please execute from sqlplus connected as user with DBA privileges
```
select OBJECT_TYPE from DBA_OBJECTS where OBJECT_NAME='DBMS_LOGMNR';
```
and send me the result.
Thanks!
Regards,
Alelsei

**Reply by iamjavadev1** [reply]
Hi averemee-si,

     I got the result
           
      OBJECT_TYPE
      ----------------
     PACKAGE
     PACKAGE BODY

Thanks!
Regards,
Iamjavadev1

**Comment by averemee-si** [comment, marked as answer]
Hm....
Synonym is missed...
Please execute from sqlplus connected as user **SYS**
```
create or replace public synonym dbms_logmnr for sys.dbms_logmnr;
```
then restart the connector.
I hope this helps.

Regards,
Aleksei
